ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml

```diff
@@ -169,7 +169,7 @@ tests:
     workflow: generic-claim
 - as: jvm-build-service-in-repo-apicurio-e2e
   optional: true
-  skip_if_only_changed: ^docs/|^\.github/|^\.tekton/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  run_if_changed: ^(tmp/build)
```

Comment:
hmm when looking at the [test job config of the project I was working on previously](https://github.com/openshift/release/blob/d3e11c3fcf31fade6c2b2d3e1f40062b1bf55e3d/ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml#L92-L113) it seems that the conditions `run_if_changed` nor `skip_if_only_changed` are not needed if you want the job to be optional (and not run by default when a PR is created) ([see this example PR where the job is not automatically triggered](https://github.com/integr8ly/integreatly-operator/pull/2878))

But I'm ok with merging this if it works too.

Reply:
interesting ... I can tell you that I went down this path @psturc because a retest in https://github.com/redhat-appstudio/jvm-build-service/pull/183 triggered the optional job, and it reminded me of what we had to do in openshift 

not seeing anything obvious in that other project ref to explain the difference

Reply:
there is one use of run if changed @psturc ... https://github.com/openshift/release/blob/master/ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml#L654